lib/order.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'menu'
+
+# Understands how to aggregate food items from a menu
```

Comment:
I like the class responsibilities - not easy to do, and they are well implemented.